config/locales/javascript/javascript.en.yml

```diff
@@ -297,7 +297,12 @@ en:
       original_post: "original post"
       result: "Result"
       count:
+        zero: "No vote so far"
         one: "1 vote so far"
         other: "<%=count%> votes so far"
+      answer_count:
+        zero: "0 vote"
```

Comment:
[The english pluralization rule](https://github.com/diaspora/diaspora/blob/develop/config/locales/cldr/plurals.rb#L12) doesn't make use of `zero`.